pkg/audit/manager.go

```diff
@@ -403,13 +414,71 @@ func (am *Manager) auditResources(
 			}
 		}
 	}
+	// loop thru each subDir in output dir to get files
+	for i := 0; i < folderCount; i++ {
+		subDir := fmt.Sprintf("%d", i)
+		pDir := path.Join(*outputDir, subDir)
+		files, err := os.ReadDir(pDir)
+		if err != nil {
+			errs = append(errs, err)
+			continue
+		}
+		for _, f := range files {
+			fileName := f.Name()
+			objNs := strings.Split(fileName, "_")[0]
+			ns := corev1.Namespace{}
+			if objNs != "" {
+				ns, err = nsCache.Get(ctx, am.client, objNs)
+				if err != nil {
+					am.log.Error(err, "Unable to look up object namespace", "objNs", objNs)
+					continue
+				}
+			}
+			contents, err := os.ReadFile(path.Join(pDir, fileName)) // #nosec G304
+			if err != nil {
+				errs = append(errs, err)
+				continue
+			}
+			objFile, err := am.readUnstructured(contents)
+			if err != nil {
+				errs = append(errs, err)
+				continue
+			}
+			augmentedObj := target.AugmentedUnstructured{
+				Object:    *objFile,
+				Namespace: &ns,
+			}
+			resp, err := am.opa.Review(ctx, augmentedObj)
+			if err != nil {
+				errs = append(errs, err)
+			} else if len(resp.Results()) > 0 {
```

Comment:
nit: does this need to be an `else`?  Can we [reduce indentation](https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow)?